docs/ballerina-by-example/examples/mutual-ssl/mutual-ssl.bal

```diff
@@ -15,9 +15,9 @@ endpoint http:ServiceEndpoint helloWorldEP {
         },
         protocols: {
             protocolName: "TLS",
-            versions: "TLSv1.2,TLSv1.1"
+            versions: ["TLSv1.2","TLSv1.1"]
```

Comment:
During the HTTP review following was suggested
User record literal for protocols
{string name, string[] version} protocol   -> protocol : {name : “TLS”, version: [“1.2”, “1.4”]}

Reply:
Fixed in : https://github.com/ballerina-platform/ballerina-lang/pull/6841